### PR TITLE
Jimmy.xu/map view2

### PR DESCRIPTION
**Pull request by YuOrii**

### CODE REVIEW

File: frontend/src/pages/MapViewPage.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        }
+        fetchData();
+    });
+
+    return (
+        <Container disableGutters maxWidth={false}>
+            {/*<Box mt={0} mb={4}>*/}
+            <Grid container spacing={3}>
+                {/* Sidebar */}
+                <Grid item xs={12} sm={4} md={3}>
```
Comment (JoyWang0801) [inline frontend/src/pages/MapViewPage.js]: We can hardcode the value to md={2.9} to eliminate the awkward gap between the navbar and the map (or any value that can avoid the overlap).
Alternatively, if time allows, we probably should find a solution to adjust this.

Comment (JoyWang0801) [reply frontend/src/pages/MapViewPage.js]: correct: md={2.6} instead of md={2.9}

File: frontend/src/pages/MapViewPage.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                    <NavBar />
+                </Grid>
+                {/* Main content */}
+                <Grid item xs={12} sm={8} md={9} container spacing={3}>
```
Comment (JoyWang0801) [inline frontend/src/pages/MapViewPage.js]: - correspondingly, here change to md={9.4}.

Comment (JoyWang0801) [reply frontend/src/pages/MapViewPage.js]: - also, we probably want to have a sx={{zIndex: 1}} or any value below than the navbar's zindex.

Comment (JoyWang0801) [reply frontend/src/pages/MapViewPage.js]: - we might want to add a Box container. looks like we are having a gap at the bottom. 
image

File: frontend/src/pages/MapViewPage.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                                    <AdvancedMarker position={{ lat: row.lat, lng: row.long }} onClick={() => setOpen(true)}>
+                                        <BubbleComponent />
+                                    </AdvancedMarker>
+                                ))}
+                                {/* <AdvancedMarker position={position} onClick={() => setOpen(true)}>
+                                    <BubbleComponent />
+                                </AdvancedMarker> */}
+                            </Map>
+                        </div>
+                    </APIProvider>
```
Comment (JoyWang0801) [inline frontend/src/pages/MapViewPage.js:56]: - add /Box here

Comment (YuOrii) [reply frontend/src/pages/MapViewPage.js:56]: Adding a /Box, under /APIProvider, crashes the page. Will leave it as is for now

**Review by Sjnaka — APPROVED** [review]

**Pull request merged**